app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListAdapter.kt

```diff
@@ -51,52 +54,83 @@ class OwnerListAdapter() : PagingDataAdapter<Solidity.Address, RecyclerView.View
                 }
             }
         }
+        if (holder is DefaultOwnerViewHolder) {
```

Comment:
can we just have 
```kotlin
 kotlin.runCatching {
                val uiModel = getItem(position)
                uiModel?.let {
                    holder.bind(it, position)
                }
            }
```
here and remove `if`. Both holders have same signature, so we shouldn't need this redundancy

Reply:
The problem is that bind is only defined in these sub classes.

Reply:
Even this doesn't compile:
``` 
if (holder is OwnerViewHolder || holder is DefaultOwnerViewHolder) {
            kotlin.runCatching {
                val uiModel = getItem(position)
                uiModel?.let {
                    holder.bind(it, position)
                }
            }
        }
```